.devcontainer/Dockerfile

```
@@ -34,6 +34,7 @@ RUN curl -fsSL -o /tmp/helm.tar.gz https://get.helm.sh/helm-${HELM_VERSION}-linu
 USER vscode
 
 RUN go install github.com/bufbuild/buf/cmd/buf@${BUF_VERSION}
+RUN go install sigs.k8s.io/kind@v0.26.0
```

I like to define all installed versions in one place as arguments at the top, so add this at the top after `HELM_VERSION`:
```
ARG KIND_VERSION=v0.26.0
```
And use it here:
```suggestion
RUN go install sigs.k8s.io/kind@${KIND_VERSION}
```

---

Done

Nice, that's a strictly superior way of doing things. I'll remember this 👍